code/__DEFINES/DNA.dm

```diff
@@ -102,12 +102,12 @@
 #define TR_HASHNAME (1<<3)
 #define TR_KEEPIMPLANTS (1<<4)
 /// changelings shouldn't edit the DNA's SE when turning into a monkey
-#define TR_KEEPSE (1<<5)
-#define TR_DEFAULTMSG (1<<6)
-#define TR_KEEPORGANS (1<<8)
-#define TR_KEEPSTUNS (1<<9)
-#define TR_KEEPREAGENTS (1<<10)
-
+#define TR_KEEPSE		(1<<5)
+#define TR_DEFAULTMSG	(1<<6)
+#define TR_KEEPORGANS	(1<<8)
+#define TR_KEEPSTUNS	(1<<9)
+#define TR_KEEPREAGENTS	(1<<10)
+#define TR_KEEPSTAMINADAMAGE (1<<11)
```

Comment:
Why are you changing the spacing of the defines?

Reply:
This was a mistake from trying to port the original pr